spec/unit/resource/action_items_spec.rb

```diff
@@ -1,74 +1,63 @@
-# frozen_string_literal: true
-require "rails_helper"
+require 'spec_helper'
 
-RSpec.describe ActiveAdmin::Resource::ActionItems do
-  let(:resource) do
-    namespace = ActiveAdmin::Namespace.new(ActiveAdmin::Application.new, :admin)
-    namespace.register(Post)
-  end
+describe ActiveAdmin::Resource::ActionItems do
 
-  describe "adding a new action item" do
-    before do
-      resource.clear_action_items!
-      resource.add_action_item :empty do
-        # Empty ...
-      end
+  [Post, "Post"].each do |resource_to_register|
+    let(:resource) do
+      namespace = ActiveAdmin::Namespace.new(ActiveAdmin::Application.new, :admin)
+      namespace.register(resource_to_register)
     end
```

Comment:
can we just have a "it allows to register resources with string"?

Reply:
done